bin/test

```diff
@@ -1,9 +1,5 @@
 #!/bin/sh
 set -exu
-LABEL='hmt'
 
-$(dirname $0)/stop
-docker run -d -p 8545:8545 --name hmtoken.ganache trufflesuite/ganache-cli:latest
-
-docker build -t $LABEL .
-docker run --net=host $LABEL npm run test
+docker-compose run hmt npm run test
```

Comment:
Where's the build.

Reply:
Should we force build every time? Added `bin/build` command now that gets run together with `bin/test` on travis.

Reply:
@posix4e the travis script will run the build command .. it'd be nice to have run only run the containers so that if the user wants to just pull a prebuilt one .. they can